Makefile

```diff
@@ -395,6 +425,15 @@ test-protobuf:
 	$(call announce-begin,"Running Protocol Buffer tests")
 	$(call announce-end,"Finished running Protocol Buffer tests")
 
+test-kaitai:
+	$(call announce-begin,"Running Kaitai Struct Python tests")
+	cd $(SWIFTNAV_ROOT) && tox -c kaitai/python/tests/tox.ini kaitai/python/tests/test_*.py
```

Comment:
Is it possible to generate a module name that would provide a more descriptive name outside of this repo?  This current layout means that we do `from python import ...` and outside of this repo this would look out of place.  Can we change the generated Python code so that imports would look like `from kaitai_sbp import ...`?  Or maybe even `from sbp.kaitai import ...`.

Reply:
I will re-arrange as follows:

`kaitai/python -> `kaitai/python/sbp`
`kaitai/perl -> `kaitai/perl/Sbp`

Reply:
:+1:

Reply:
Done